compatibility.json

```diff
@@ -1,26 +1,26 @@
 {
   "0.15.0": {
       "nns": {
-          "versions": ["0.1.0", "0.2.0"]
+          "versions": ["0.2.0"]
```

Comment:
as of this moment, the release-0.15.0 branch does not support the new filenames

Reply:
should we first update release-0.15.0 then?

Reply:
Is there an order for doing things that isn't breaking? 
If not, can we make it so by having one extension release have both filename formats?